rust/Makefile

```diff
@@ -46,7 +46,7 @@ $(objtree)/rust/bindings_generated.rs: $(srctree)/rust/kernel/bindings_helper.h
 quiet_cmd_exports = EXPORTS $@
       cmd_exports = \
 	$(NM) -p --defined-only $< \
-		| grep -F ' T ' | cut -d ' ' -f 3 | grep -E '^(__rust_|_R)' \
+		| grep -E '( T | R )' | cut -d ' ' -f 3 | grep -E '^(__rust_|_R)' \
```

Comment:
Which symbol triggered this? The generated `$ops`?

Reply:
Yes, exactly.